docs/troubleshooting.md

```diff
@@ -515,22 +515,51 @@ The Node.js runtime of the
 [App Engine standard environment](https://cloud.google.com/appengine/docs/standard/nodejs/)
 comes with all system packages needed to run Headless Chrome.
 
-To use `puppeteer`, simply list the module as a dependency in your
-`package.json` and deploy to Google App Engine. Read more about using
-`puppeteer` on App Engine by following
-[the official tutorial](https://cloud.google.com/appengine/docs/standard/nodejs/using-headless-chrome-with-puppeteer).
+To use `puppeteer`, specify the module as a dependency in your `package.json`
+and then override the puppeteer cache directory by including a file named 
+`.puppeteerrc.cjs` at the root of your application with the contents:
+
+```ts
+const {join} = require('path');
+
+/**
+ * @type {import("puppeteer").Configuration}
+ */
+module.exports = {
+  cacheDirectory: join(__dirname, 'node_modules', '.puppeteer_cache'),
```

Comment:
`.cache` folder is commonly used to signify that a package wants to store some data, so let's use that one.
```suggestion
  cacheDirectory: join(__dirname, 'node_modules', '.cache', 'puppeteer'),
```